tests/infrastructure/test_ialirt_ingest_lambda_stack.py

```diff
@@ -0,0 +1,33 @@
+"""Test the IAlirt database stack."""
+
+import pytest
+from boto3.dynamodb.conditions import Key
+
+
+@pytest.fixture()
+def populate_table(table):
+    """Populate DynamoDB table."""
+    items = [
+        {
+            "reset_number#met": "0#2025-07-11T12:34:56Z",
```

Comment:
MET should be an integer value
```suggestion
            "reset_number#met": "0#123",
```